ext/spl/spl_heap.c

```diff
@@ -267,7 +267,7 @@ static spl_ptr_heap *spl_ptr_heap_init(spl_ptr_heap_cmp_func cmp, spl_ptr_heap_c
 /* }}} */
 
 static void spl_ptr_heap_insert(spl_ptr_heap *heap, void *elem, void *cmp_userdata) { /* {{{ */
-	int i;
+	size_t i;
```

Comment:
Can you move this into the for loop? As that's the only place it is used

Reply:
It is used by spl_heap_elem_copy() at the very end of the function

Reply:
It's also accessed in line 295. But it would be possible to move it right before the loop (or rename it into something more reasonable).

Reply:
Time to refresh my knowledge then 😄 Thanks, I also updated var names to be a bit more descriptive.